### DISCUSSION TITLE
No 2FA  leads to "Account Restrictions". But what are they exactly?

### DISCUSSION THREAD

**Discussion by CyberFoxHax** [community/community#67053] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Several times has be 2FA deadline been moved. First it was march, then it's early September, now it's late September.

My hope is; upon a login you'll be taken you to the 2FA setup page. And that will be all you can do with your account.
I somehow doubt that Github would heartless enough to effectively "ban" non-compliers.

What do you suppose will happen when the deadline is reached?

**Comment by kuroodo** [comment, marked as answer]
Essentially what you said. Your account will be locked until you enable 2FA. But I wouldn't put it past Microsoft at this point to delete the accounts of non-compliers after a few months.

My account is going to remain locked until they either stop forcing 2FA, or allow me to set up 2FA with email. They won't be getting any more money from me either. I've migrated to GitLab.

**Reply by CyberFoxHax** [reply]
completely agree. I migrated my shit to gitlab this morning.

While I think the 2FA solution doesn't seem to be problematic on it's own. It's offline, open source, decentralized and all the good stuff. I've done the research already. FreeOTP+ is a good FOSS app. I can put the key into my Keepass chain too as a redundancy. I might implement this within my own company.

But now that they've said I must, I wont. We'll see who blinks first.

(Oh and it seems SSH keys will be unaffected, so i can still push and pull like normal)

**Reply by landley** [reply]
Can I still delete my account after the 2fa lockdown, or do I need to delete it before I lose access?

**Reply by kjkrum** [reply]
@landley I deleted all my repos and updated my profile so it's just a signpost to my Gitlab. If I can no longer write bug reports, so be it. Github is kill.
